v0.7/medical_imaging/3d-unet/Makefile

```diff
@@ -33,8 +33,10 @@ POSTPROCESSED_DATA_DIR := $(BUILD_DIR)/postprocessed_data
 MODEL_DIR := $(BUILD_DIR)/model
```

Comment:
It looks like you need to add a `mkdir $MODEL_DIR` otherwise the script fails if there is no existing $(BUILD_DIR)/model

Reply:
good point. Will fix this

Reply:
fixed